src/views/Header.js

```diff
@@ -119,25 +120,35 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
     if (props.scene.index === 0) {
       return null;
     }
-    const backButtonTitle = this._getBackButtonTitleString(props.scene);
-    const truncatedBackButtonTitle = this._getTruncatedBackButtonTitle(
-      props.scene
-    );
     const width = this.state.widths[props.scene.key]
       ? (this.props.layout.initWidth - this.state.widths[props.scene.key]) / 2
       : undefined;
+    var backButtonElement = options.headerBack;
```

Comment:
I think it would be better to extract the logic that determines `backButtonElement` into a function that returns the value of what `backButtonElement` should be rather than imperatively determining what its value is. See `this._getBackButtonTitleString` as an example.